src/lib/journey/stages/generic.svelte

```diff
@@ -41,36 +41,38 @@
   }
 </script>
 
+{#if displayIcon}
+  <div class="tw_flex tw_justify-center">
+    <ShieldIcon classes="tw_text-gray-400 tw_fill-current" size="72px" />
+  </div>
+{/if}
+<h1 class="tw_primary-header dark:tw_primary-header_dark">
+  <!-- TODO: Needs localization strategy -->
+  <Sanitize html={true} string={step?.getHeader() || ''} />
+</h1>
+<p
+  class="tw_text-center tw_-mt-5 tw_mb-2 tw_py-4 tw_text-secondary-dark dark:tw_text-secondary-light"
+>
+  <!-- TODO: Needs localization strategy -->
+  <Sanitize html={true} string={step?.getDescription() || ''} />
+</p>
+
```

Comment:
Trying to "flatten" this component out a bit. Moving some of the conditionals (checking for `step`) into the journey component allowed for this.